libs/block-editor/src/lib/extensions/components/suggestions/suggestions.component.ts

```diff
@@ -57,10 +57,12 @@ export class SuggestionsComponent implements OnInit, AfterViewInit {
 
     initialItems: DotMenuItem[];
     isFilterActive = false;
+    currentLanguage = DEFAULT_LANG_ID;
```

Comment:
Let's make this an `@Input` just to make it explicit that we change it from outsite.